tests/unit/Entity/EntityTest.php

```diff
@@ -14,21 +14,20 @@
  * @deprecated
  * This test class is to be phased out, and should not be used from outside of the component!
  *
- * @group Wikibase
- * @group WikibaseDataModel
+ * @covers Wikibase\DataModel\Entity\Entity
  *
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  * @author Daniel Kinzler
  */
-abstract class EntityTest extends \PHPUnit_Framework_TestCase {
```

Comment:
We can also just remove the test in this patch already. It doesn'r really matter anyways. However, I didn't want to leave `Entity` untested and thus made the test non-abstract so that it tests a `Entity` implementation instead.